subprojects/core/src/main/java/org/gradle/internal/buildevents/BuildLoggerFactory.java

```diff
@@ -41,6 +41,6 @@ public BuildLoggerFactory(StyledTextOutputFactory styledTextOutputFactory, WorkV
     }
 
     public BuildLogger create(Logger logger, LoggingConfiguration loggingConfiguration, BuildStartedTime buildStartedTime, BuildRequestMetaData buildRequestMetaData) {
-        return new BuildLogger(logger, styledTextOutputFactory, loggingConfiguration, buildRequestMetaData, buildStartedTime, clock, workValidationWarningReporter, gradleEnterprisePluginManager);
+        return new BuildLogger(logger, styledTextOutputFactory, loggingConfiguration, buildRequestMetaData, buildStartedTime, clock, workValidationWarningReporter, gradleEnterprisePluginManager) {};
```

Comment:
❌  I don't see why this would be a necessary addition.

Reply:
Only a typo. Removed.